content/working_with/console/pages/_index.md

```diff
@@ -22,7 +22,7 @@ Below are the default set of pages (templates) presented per Cloudify version an
 ***main-sys_admin*** template applies to `sys-admin` users in all the tenants, and includes the following pages:
 
 * **Dashboard** - An overview of the existing executions and system statistics.
-* **Cloudify Catalog** - Contains Plugins Catalog showing available Cloudify plugins and Blueprints Catalog presenting blueprint examples.
+* **{{< param product_name >}} Catalog** - Contains Plugins Catalog showing available {{< param product_name >}} plugins and Blueprints Catalog presenting blueprint examples.
```

Comment:
It means that on our (Stage) side we need to make name of the page to be dynamic, right?

Reply:
This is my bad. I will revert this line.